app/src/minimal/kotlin/io/homeassistant/companion/android/launch/LaunchPresenterImpl.kt

```diff
@@ -3,22 +3,24 @@ package io.homeassistant.companion.android.launch
 import io.homeassistant.companion.android.BuildConfig
 import io.homeassistant.companion.android.common.data.integration.DeviceRegistration
 import io.homeassistant.companion.android.common.data.servers.ServerManager
+import io.homeassistant.companion.android.notifications.PushManager
 import javax.inject.Inject
 import kotlinx.coroutines.launch
 import timber.log.Timber
 
 class LaunchPresenterImpl @Inject constructor(
     view: LaunchView,
-    serverManager: ServerManager
-) : LaunchPresenterBase(view, serverManager) {
+    serverManager: ServerManager,
+    pushManager: PushManager
+) : LaunchPresenterBase(view, serverManager, pushManager) {
```

Comment:
Now that you have an abstraction you could potentially already remove the two version of the LaunchPresenterImpl and keep only the one from full. Let's avoid as much as possible duplicates. It makes maintenance of minimal easier.